cypress/support/step_definitions/testSetup.md

```diff
@@ -1,15 +1,50 @@
 # TestSetup Glue
-## I initialize a {string} test
 
-### Purpose: initalize a test during prerequisite
+## the environment has been set up for {string} tests
+
+### Purpose: Sets up the environment for the specified test based on the test name. 
+
+#### Below are the tasks that are performed based on the test name:
+- This glue will used to initialize the firebolt object for the dynamic calls.
+  - Passing the key name in `module:method` format
+    ##### Format
+    ```javascript
+    Given the environment has been set up for 'module:method' tests
+    ```
+    - In the abover format, `module` is the name of the Firebolt object and firebolt object fetched and stored in the `runtime` environment variable using module name.
+    - `module:method` this will split the key name by `:` and both the module and method will be stored in the `runtime` environment variable.
+  - Example: `Given the environment has been set up for 'Localization:locale' tests` - `Localization` is the firebolt object name.
+
+  - Passing the key name directly
+    ##### Format
+    ```javascript
+    Given the environment has been set up for 'keyName' tests
+    ```
+    - In the above format, `keyName` is the name of the Firebolt object and firebolt object fetched and stored in the `runtime` environment variable using key name.
+    
+    Example: `Given the environment has been set up for 'ADVERTISING_SKIPRESTRICTION' tests` - `ADVERTISING_SKIPRESTRICTION` is the firebolt object name.
+
+  - Passing the key name with spaces
+    ##### Format
+    ```javascript
+    Given the environment has been set up for 'key Name' tests
+    ```
+    - In the above format, If `key Name` having spaces, then it should be replaced with `_` in the key name and finally, the key name converted to uppercase.
+    
+    Example: `Given the environment has been set up for 'Advertising skiprestriction' tests` - `ADVERTISING_SKIPRESTRICTION` is the firebolt object name.
+
```

Comment:
In the above format, If key Name having spaces, then it should be replaced with _ in the key name and finally, the key name converted to uppercase.

is not correct right  ? 

In the above format, If key Name is having spaces, then it will be replaced with _ in the key name and finally, the key name converted to uppercase.

Reply:
yes, corrected